posts/2022/02/what-do-we-need-to-write.md

```diff
@@ -0,0 +1,126 @@
+- Being responsible for what you say on a company blog can be stressful.
+- If I haven't looked at my drafts in a while, I don't usually find anything 
+there that I feel is worth continuing on.
+
+### Maybe it's not so bad
+
+Some of the authors' favorite parts of these lists is that many of the 
+obstacles complement each other! Just like there isn't one single kind of 
+author, there isn't one single set of needs people have in order to write. Do 
+we need people or do we need alone time? Do we need the force of deadlines or 
+the relaxation of an open-ended timeline? One of our team members even said
+
+> It's interesting to see how everyone is struggling in different ways.
+
+Bring those thoughts with you while we discuss the next question…
+
+## How do people like to write?
+
+Sometimes writings do successfully get published! 🎉 We asked our team about 
+what goes right when writing does get completed as planned. People 
+commented that writing works best for them when the following happens:
+
+- Knowing who you are writing for is important. For practice, folks suggested 
+writing for yourself or close allies.
+- Writing about topics you like (or at least tying another topic into that).
+- I like to write as part of a organized meet up where you critique and share 
+each other's work with the intent of publishing at the end. ([Writer's workshop style](https://github.com/Quansight/writers-workshop))
+- Having a good editor / someone to provide constructive critiques is so helpful.
+- Co-authoring is the best!
+- As a way to bring people into new projects.
+- It's so helpful to be hyped and have that support system.
+- To by hyped by others in the form of "I double-doggy-dare you" or "I bet a 
+dollar you can."
+
+The "I published a blog post in the last few months" team also provided some 
+writing lifestyle advice:
+
+- Write in a cabin in the woods (warning: you may risk hiking instead of 
+writing)
+- Drink tea while writing.
+- A legal pad and a computer go together. No computer writing alone.
+- Relax while writing. You'll think better. (When there's no deadline!)
+
+## What are we to do?
+
+Write! Write for yourself. Write for each other. 
+
+To do this sustainably, we need to experiment with techniques that shorten the 
+gap between the obstacles and all the good feelings listed above.
+
+One pattern we discovered in this discussion is that people agree writing (and 
+publishing) takes a team. Whether people struggled at the beginning or at the end of 
+the process, everyone mentioned needing support at one time or another. And of 
+course, working with others is better when you find a way to have fun. Creating 
+a system to engage in fun, consistent collaboration may be the first step to helping all our writers make it across the finish line.
+
+In need of encouragement? Remember the wise words of another team member:
+
+> I can't wait to read what you want to write.
```

Comment:
If that's an actual quote, then ignore this comment, but I'd put "I can't wait to read what you write."

Reply:
It says it was a quote in the notes, but we can ask @tonyfast how he'd like to be (secretly) quoted!